andela/wg-fortem
# 166152326 Allows comparison of members

#### Title

This PR allows for the comparison of members' weight data

#### Description



When two users are training together it would be useful to see a comparison of some sorts. Two graphs showing the users' weight data gives a good visual impression of the comparison

#### Type of change








#### How Can This Be Manually Tested?
To be able to test the feature, check out the branch `story/166152326-comparison-of-members` and then `python manage.py runserver`.

- Login as a gym manager

- Navigate to the URL `http://127.0.0.1:8000/en/gym/<gym_id>/members` on your browser

![image](https://user-images.githubusercontent.com/44572764/59695476-f6308980-91f2-11e9-89c5-80b7f0d9887d.png)

- Select 2 users

![image](https://user-images.githubusercontent.com/44572764/59695493-fdf02e00-91f2-11e9-95f6-4e3794f6becf.png)

- Compare

![image](https://user-images.githubusercontent.com/44572764/59695394-d731f780-91f2-11e9-8a9d-bb8957236742.png)

#### Acceptance Criteria: 



#### Checklist:
- [x] Adds a means to compare weight data of 2 members
- [x] Assigns comparison functionality to the gym manager

#### PT stories
[#166152326](https://www.pivotaltracker.com/story/show/166152326)
